src/swagger/api.yaml

```diff
@@ -11818,6 +11827,10 @@ definitions:
         web: '#/definitions/WebUserFactor'
         webauthn: '#/definitions/WebAuthnUserFactor'
       propertyName: factorType
+  UserFactorProfile:
+    properties: {}
```

Comment:
Are there any common properties that can be moved here like `credentialId` ?
(note: i'm not sure if that is common across all of the factor profiles)

Reply:
Actually, there is no commonality between the individual user factor profiles. Like we discussed on Slack, I will remove this from the spec and have the code copy pasted into the project.